docs/serving/graph-ha-cfg.md

```diff
@@ -21,6 +21,10 @@ performs additional processing of events after the function handler returns, the
 lost. The mechanism of Window ACK provides a solution for such 
 stateful event processing.
 
+```{admonition} Note
+For statefeul functions, each worker has its own state.
```

Comment:
Maybe mention `MapWithState` as an example of this.

Reply:
added